### ISSUE TITLE
Stopped working after latest windows 11 preview (build 22523.1000)

### ISSUE THREAD

**Issue by jelical** [DDoSolitary/LxRunOffline#188]
Example output: 
```
D:\>LxRunOffline.exe rg -n main -d d:\main
[ERROR] Couldn't get the value "DistributionName" of the registry key "Software\Microsoft\Windows\CurrentVersion\Lxss\TryStoreWSL".
Reason: The system cannot find the file specified.
```

**Comment by a55uka** [comment]
Are you sure that the installation is named 'main'

**Comment by Evenfire** [comment]
Hi,

I just installed `LxRunOffline` (3.5.0, using chocolatey) on a machine running build 22523.1000 and I get the same error when running the following command:
```
PS C:\WINDOWS\system32> LxRunOffline.exe list
[ERROR] Couldn't get the value "DistributionName" of the registry key "Software\Microsoft\Windows\CurrentVersion\Lxss\AppxInstallerCache".
Reason: The system cannot find the file specified.
```

The command `LxRunOffline.exe export` raises the same error.
But I'm able to use the command `LxRunOffline.exe get-default`.

EDIT:

Here is the registry's content

```
PS C:\WINDOWS\system32> (Get-ChildItem "HKCU:\Software\Microsoft\Windows\CurrentVersion\Lxss" -Recurse)


    Hive: HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Lxss


Name                           Property
----                           --------
AppxInstallerCache
{acddf79a-be23-48ba-b91a-af33f State             : 1
a45b798}                       DistributionName  : Ubuntu-18.04
                               Version           : 2
                               BasePath          : C:\Users\boby\AppData\Local\Packages\CanonicalGroupLimited.Ubuntu18.04onWindows_79rhkp1fndgsc\LocalState
                               Flags             : 15
                               DefaultUid        : 1000
                               PackageFamilyName : CanonicalGroupLimited.Ubuntu18.04onWindows_79rhkp1fndgsc
{c2000117-ca36-4a29-a7e8-b4d21 State            : 1
4d90c85}                       DistributionName : docker-desktop-data
                               Version          : 2
                               BasePath         : \\?\C:\Users\boby\AppData\Local\Docker\wsl\data
                               Flags            : 15
                               DefaultUid       : 0
{c574ed4b-590e-42e6-a9f5-044a7 State            : 1
1f1caf2}                       DistributionName : docker-desktop
                               Version          : 2
                               BasePath         : \\?\C:\Users\boby\AppData\Local\Docker\wsl\distro
                               Flags            : 15
                               DefaultUid       : 0
```

**Comment by jelical** [comment]
> Are you sure that the installation is named 'main'

Yes, it called main

**Comment by edgimar** [comment]
[This issue](https://github.com/DDoSolitary/LxRunOffline/issues/171) is probably the same one -- it is fixed, but the fix is not in the latest release.  We just need an updated release (newer than 3.5.0) -- or use a development release from [here](https://ddosolitary-builds.sourceforge.io/LxRunOffline/).

**Comment by martin-rueegg** [comment]
> [This issue](https://github.com/DDoSolitary/LxRunOffline/issues/171) is probably the same one -- it is fixed, but the fix is not in the latest release. We just need an updated release (newer than 3.5.0) -- or use a development release from [here](https://ddosolitary-builds.sourceforge.io/LxRunOffline/).

Slightly off-topic, but your link does (currently not work). Also on the [main files page](https://sourceforge.net/projects/lxrunoffline.mirror/files/) are no development releases available. Anything you can do about it? Thanks!

**Comment by edgimar** [comment]
@martin-rueegg, you're right - the link appears to have become broken over the last year or so.  There's an archived version [here](https://web.archive.org/web/20221007093931/https://ddosolitary-builds.sourceforge.io/LxRunOffline/), but it might be older than the version I had referenced.

**Comment by martin-rueegg** [comment]
> @martin-rueegg, you're right - the link appears to have become broken over the last year or so. There's an archived version [here](https://web.archive.org/web/20221007093931/https://ddosolitary-builds.sourceforge.io/LxRunOffline/), but it might be older than the version I had referenced.

Thanks, @edgimar 

Unfortunately, it is only showing the list of files that _were_ available at the time of the snapshot. However, the files themselves have not been captured, of course. So the latest shown release cannot be downloaded, nor 
- here: https://ddosolitary-builds.sourceforge.io/LxRunOffline/LxRunOffline-v3.5.0-33-gbdc6d7d-mingw.zip, nor
- here: https://web.archive.org/web/20221007093931/https://ddosolitary-builds.sourceforge.io/LxRunOffline/LxRunOffline-v3.5.0-33-gbdc6d7d-mingw.zip

Should I create a new issue regarding the downloads of the development binaries?

**Comment by edgimar** [comment]
@martin-rueegg, sure - feel free to create a separate issue.

**Comment by MikhailLipanin** [comment]
@martin-rueegg, Have you find a solution for this problem?

**Comment by martin-rueegg** [comment]
@MikhailLipanin see here: https://github.com/DDoSolitary/LxRunOffline/issues/204#issue-1644251017. However, the files mentioned there still have the same issue.

**Comment by TBOZ31** [comment]
I overcame this issue by simply deleting the registry key "TryStoreWSL" in the registry under HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Lxss

**Comment by TullyMonster** [comment]
> I overcame this issue by simply deleting the registry key "TryStoreWSL" in the registry under HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Lxss

Thank you. That seems to work.
